### PR TITLE
Add repo tests

### PR DESCRIPTION
**Pull request by yuna878**
Add test cases for repo methods.

### CODE REVIEW

File: test/repos/poll.test.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ import UsersRepo from '../../src/repos/UsersRepo';
 import dbConnection from '../../src/db/DbConnection';
 
 let session;
+let session2;
 let id;
```
Comment (young-k) [inline test/repos/poll.test.js]: can we alphabetize these, so `id` before `session`

File: test/repos/draft.test.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ let user1;
 let user2;
 let draft1;
 let draft2;
```
Comment (young-k) [inline test/repos/draft.test.js]: can we alphabetize these, so `draft` before `user`

File: test/repos/session.test.js [hunk 3]
```diff
@@ -18,112 +23,198 @@ beforeAll(async () => {
...
+    let polls = await SessionsRepo.getPolls(id);
+    expect(polls.length).toEqual(0);
+
+    const poll = await PollsRepo.createPoll('Poll', session, {}, true, 'MULTIPLE_CHOICE');
+    const poll2 = await PollsRepo.createPoll('', session, {}, false, 'FREE_RESPONSE', {});
+    polls = await SessionsRepo.getPolls(id);
+    expect(polls.length).toEqual(2);
+
+    await PollsRepo.deletePollById(poll.id);
+    await PollsRepo.deletePollById(poll2.id);
```
Comment (young-k) [inline test/repos/session.test.js]: Nifty trick, you can do `await Promise.all([PollsRepo.deletePollById(poll.id), PollsRepo.deletePollById(poll2.id)]);`

Not sure if it's actually cleaner here, but just something to keep in mind! @meganle what do you think of doing that here?

File: test/repos/draft.test.js [hunk 4]
```diff
@@ -19,30 +20,47 @@ beforeAll(async () => {
...
     expect(draft1.text).toBe('What\'s up?');
     expect(draft1.options).toEqual(['A', 'B', 'C', 'D']);
     expect(draft1.user).toEqual(user1);
+
+    draft2 = await DraftsRepo.createDraft('Are you awake?', [], user1);
     expect(draft2.text).toBe('Are you awake?');
     expect(draft2.options).toEqual([]);
     expect(draft2.user).toEqual(user1);
+
+    draft3 = await DraftsRepo.createDraft('',['AB'], user1);
```
Comment (meganle) [inline test/repos/draft.test.js]: space after the first comma

File: test/repos/session.test.js [hunk 5]
```diff
@@ -18,112 +23,198 @@ beforeAll(async () => {
...
 
-test('Get Members Of Session', async () => {
+test('Get Members of Session', async () => {
     const members = await SessionsRepo.getUsersBySessionId(id, 'member');
     expect(members.length).toEqual(1);
     expect(members[0].googleId).toBe(user2.googleId);
+
+    const members2 = await SessionsRepo.getUsersBySessionId(id2, 'member');
+    expect(members2.length).toEqual(2);
+    expect(members2[0].id).toBe(user3.id)
```
Comment (meganle) [inline test/repos/session.test.js]: semicolon

File: test/repos/session.test.js [hunk 6]
```diff
@@ -18,112 +23,198 @@ beforeAll(async () => {
...
 });
 
-test('Get Members And Admins Of Session', async () => {
+test('Get All Users of Session', async () => {
     const users = await SessionsRepo.getUsersBySessionId(id);
     expect(users.length).toEqual(2);
     expect(users[0].id).toBe(user.id);
     expect(users[1].id).toBe(user2.id);
+
+    const users2 = await SessionsRepo.getUsersBySessionId(id2);
```
Comment (meganle) [inline test/repos/session.test.js]: unused variable?

File: test/repos/session.test.js (hunk 3 above)
Comment (meganle) [reply test/repos/session.test.js]: Interesting, yeah not sure if it's cleaner. Personally, i think having them on separate lines is more readable

**Pull request merged**